platform-sdk/swirlds-common/src/main/java/com/swirlds/common/wiring/wires/output/NoOpOutputWire.java

```diff
@@ -0,0 +1,124 @@
+import edu.umd.cs.findbugs.annotations.NonNull;
+import java.util.function.Consumer;
+import java.util.function.Function;
+import java.util.function.Predicate;
+
+/**
+ * An output wire that doesn't actually do anything. When asked to solder to another wire, it does nothing. When asked
+ * when asked to build a transformer (or variations there upon) it produces no-op implementations.
+ *
+ * @param <OUT> the type of data passed to the forwarding method
+ */
+public class NoOpOutputWire<OUT> extends StandardOutputWire<OUT> {
+
+    /**
+     * Constructor.
+     *
+     * @param model the wiring model containing this output wire
+     * @param name  the name of the output wire
+     */
+    public NoOpOutputWire(@NonNull final StandardWiringModel model, @NonNull final String name) {
+        super(model, name);
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    protected void addForwardingDestination(@NonNull final Consumer<OUT> destination) {
+        // intentional no-op
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public void forward(@NonNull final OUT data) {
+        // intentional no-op
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public void solderTo(@NonNull final InputWire<OUT> inputWire, @NonNull final SolderType solderType) {
+        // intentional no-op
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public void solderTo(
+            @NonNull final String handlerName,
+            @NonNull final String inputWireLabel,
+            final @NonNull Consumer<OUT> handler) {
+        // intentional no-op
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @NonNull
+    @Override
+    public OutputWire<OUT> buildFilter(
+            @NonNull final String filterName,
+            @NonNull final String filterInputName,
+            @NonNull final Predicate<OUT> predicate) {
+        return new NoOpOutputWire<>(getModel(), filterName);
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @NonNull
+    @Override
+    public <ELEMENT> OutputWire<ELEMENT> buildSplitter(
+            @NonNull final String splitterName, @NonNull final String splitterInputName) {
+        return new NoOpOutputWire<>(getModel(), splitterName);
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @NonNull
+    @Override
+    public <NEW_OUT> OutputWire<NEW_OUT> buildTransformer(
+            @NonNull final String transformerName,
+            @NonNull final String transformerInputName,
+            @NonNull final Function<OUT, NEW_OUT> transformer) {
+        return new NoOpOutputWire<>(getModel(), transformerName);
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @NonNull
+    @Override
+    public <NEW_OUT> OutputWire<NEW_OUT> buildAdvancedTransformer(
+            @NonNull final AdvancedTransformation<OUT, NEW_OUT> transformer) {
+        return new NoOpOutputWire<>(getModel(), "this name doesn't matter because this wire doesn't do anything");
```

Comment:
You used the actual `transformerName` for the other noop methods - the matching pattern here would be to use `transformer.getTransformerName()`

Up to you, though, since it doesn't matter, as stated

Reply:
Changed as suggested.